.github/workflows/graph-toolkit-released.yml

```diff
@@ -26,4 +26,19 @@ jobs:
           subject="Microsoft Graph Toolkit v${{ github.event.client_payload.version }} Released"
 
           echo "to=$emails" >> $GITHUB_OUTPUT
-          echo "subject=$subject" >> $GITHUB_OUTPUT
\ No newline at end of file
+          echo "subject=$subject" >> $GITHUB_OUTPUT
```

Comment:
why not use the send-email-report action?

Reply:
Their commands are different, no double quotation marks for them. It is html not only string using "body": ${{ steps.detail.outputs.result }},

Reply:
The original stopping email PR as reference: https://github.com/OfficeDev/TeamsFx/pull/9481/files#diff-64fade0467319bab9b796ca00c34fcb10fcd65d42e7f461aa6cc688898762de9L56

Reply:
updated to use send-email-report action